<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaConsultar</class>
 <widget class="QWidget" name="VentanaConsultar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consultar datos</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="consultarRutas">
      <attribute name="title">
       <string>Rutas de un tren</string>
      </attribute>
      <widget class="QPushButton" name="btConsR_3">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>360</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Consultar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btSalir_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>370</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtCodTTren_3">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>80</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_49">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>10</y>
         <width>81</width>
         <height>36</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RUTAS</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtRutas_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>501</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_47">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>120</y>
         <width>270</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ingrese el código del tren: </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_48">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>270</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ingrese el código del tipo de tren: </string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtCodTren_3">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>120</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="consultarPrecio">
      <attribute name="title">
       <string>Precio de una ruta</string>
      </attribute>
      <widget class="QPushButton" name="btConsPre_2">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>380</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Consultar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>120</y>
         <width>270</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ingrese el código del tren: </string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtR">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>150</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="btSalir_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>380</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtConsPrec_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>501</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>270</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ingrese el código de la ruta: </string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtT">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>110</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtTT">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>70</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>20</y>
         <width>91</width>
         <height>36</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PRECIO</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>270</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ingrese el código del tipo de tren: </string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="consultarCiudades">
      <attribute name="title">
       <string>Ciudades</string>
      </attribute>
      <widget class="QTextEdit" name="textEdit_5">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>501</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btCancelarTiempo_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>370</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btCancelarPrecio_7">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>360</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Consultar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>10</y>
         <width>121</width>
         <height>36</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CIUDADES</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtCodPaisSalida_7">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>80</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>270</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ingrese el código del país: </string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="consultarConexiones">
      <attribute name="title">
       <string>Conexiones</string>
      </attribute>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>10</y>
         <width>151</width>
         <height>36</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CONEXIONES</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btCancelarPrecio_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>370</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>270</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ingrese el código del país: </string>
       </property>
      </widget>
      <widget class="QPushButton" name="btCancelarPrecio_6">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>370</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Consultar</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_4">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>170</y>
         <width>501</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtCodPaisSalida_5">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>80</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="consultarTrenes">
      <attribute name="title">
       <string>Trenes</string>
      </attribute>
      <widget class="QPushButton" name="btSali2_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>370</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnConsT_3">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>360</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Consultar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_51">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>270</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ingrese el código del tipo de tren: </string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtConsultarT_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>501</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_50">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>10</y>
         <width>91</width>
         <height>36</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TRENES</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtCodT_5">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>80</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HP Simplified Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btSalir_8</sender>
   <signal>clicked()</signal>
   <receiver>VentanaConsultar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btSalir_4</sender>
   <signal>clicked()</signal>
   <receiver>VentanaConsultar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancelarTiempo_2</sender>
   <signal>clicked()</signal>
   <receiver>VentanaConsultar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancelarPrecio_3</sender>
   <signal>clicked()</signal>
   <receiver>VentanaConsultar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btSali2_3</sender>
   <signal>clicked()</signal>
   <receiver>VentanaConsultar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
